### PR TITLE
prevent two sessions in close succession, time interval = 5 sec.

### PR DESCRIPTION
**Pull request by dzhuang**
Works when JS is disabled, and when the student is not intended to multiple post the requests.

### CODE REVIEW

**Comment by inducer** [comment]
Thanks for coding that up and submitting it!

To be properly race-free (i.e. protected by a `serializable`-level transaction), the query for `past_sessions` should be within `post_start_flow`. I also wonder if we should make that value (5 seconds) configurable, such as by: `getattr(settings, "RELATE_SESSION_RESTART_COOLDOWN_SECONDS", 10)`.

**Comment by dzhuang** [comment]
Thank you for your suggestions. I've update the code as suggested.

File: relate/settings.py [hunk 1]
```diff
@@ -164,6 +164,8 @@
 
 RELATE_ADMIN_EMAIL_LOCALE = "en_US"
 
+RELATE_SESSION_RESTART_COOLDOWN_SECONDS = 10
```
Comment (inducer) [inline relate/settings.py]: Because we're using `getattr()` with a default, this isn't needed. I've gone ahead and removed it.

**Comment by inducer** [comment]
Looks good. Thanks!

**Pull request merged**